srcpkgs/trident-icons/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'trident-icons'
+pkgname=trident-icons
+version=2019.10.12
+revision=1
+short_desc="Icon themes from Project Trident"
```

Comment:
`archs=noarch`

Reply:
Just pushed this archs=noarch fix into the PR a moment ago.
Thanks for pointing this out!